### PR TITLE
feat!: move to microgenerator

### PR DESCRIPTION
**Pull request by arithmetic1728**

### CODE REVIEW

File: synth.py [hunk 1]
```diff
@@ -35,146 +35,27 @@
...
-    Attributes:''',
-)
-
-# Fix accesscontextmanager and orgpolicy imports
-s.replace(
-    "google/cloud/asset_v*/types.py",
-    "from google\.cloud\.asset_(v.*)\.proto import ((access_level_pb2)|(service_perimeter_pb2)|(access_policy_pb2))",
-    "from google.identity.accesscontextmanager.v1 import \g<2>",
-)
-
```
Comment (busunkim96) [inline synth.py:115]: Did you see any errors related to accesscontextmanager or orgpolicy? It's great if they "just work" now, I'm just surprised.

File: synth.py [hunk 2]
```diff
@@ -35,146 +35,27 @@
...
-# Glue in Project Path Method.
-# TODO: Remove during microgenerator transition
-count = s.replace(
-    [
-        "google/cloud/asset_v1/gapic/asset_service_client.py",
-        "google/cloud/asset_v1beta1/gapic/asset_service_client.py",
-    ],
-    "(def __init__\()",
-    '''@classmethod
-    def project_path(cls, project):
```
Comment (busunkim96) [inline synth.py:131]: Can you note that the `project_path` method has been removed in the migration guide?

File: synth.py (hunk 1 above)
Comment (arithmetic1728) [reply synth.py:115]: I don't see any errors, they just worked :)

File: synth.py (hunk 2 above)
Comment (arithmetic1728) [reply synth.py:131]: done

**Comment by arithmetic1728** [comment]
> LGTM, but please hold on merging until the samples have been incorporated.

sure

**Comment by arithmetic1728** [comment]
@busunkim96 I completed migrating the samples, please take another look. Let's release 1.4.0 first (https://github.com/googleapis/python-asset/pull/61) for the last 1.x release. I already updated the last version supporting python 2.7 to 1.4.0 in README.rst, so we can merge this PR after that.

**Pull request merged**